docs/reference/verify-assets.mdx

```diff
@@ -305,7 +302,181 @@ $ sudo ig daemon --verify-image=false
 ...
 # Switch to another terminal
 $ gadgetctl run ghcr.io/your-repo/gadget/trace_open:latest
-WARN[0000] your_ip              | image signature verification is disabled due to using corresponding option
+WARN[0000] your_ip              | gadget signature verification is disabled due to using corresponding option
+RUNTIME.CONTAINERN… TIMESTAMP  PID        UID        GID        MNTNS_ID   ERR        FD         FLAGS      MODE      COMM      FNAME
+```
+
+</TabItem>
+</Tabs>
+
+### Verify with notation
+
+Along cosign, Inspektor Gadget also supports running gadgets signed with notation.
+The main difference consist in using certificates and a [policy document](https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#trust-policy).
+Note that, notation and cosign verification are complementary and it is possible to run gadgets signed with both method.
+
+<Tabs groupId="env">
+<TabItem value="kubectl-gadget" label="kubectl gadget">
+Compared to `ig`, you cannot specify information with regard to verification when calling `kubectl gadget run`:
```

Comment:
I know it was there before, but I don't see any value on this. Just provide the good way to do this instead of explaining why the other way doesn't work, it's confusing.

The "When using Inspektor Gadget on Kubernetes" sentence below is enough IMO